config/customcss/orderly-web.yml

```diff
@@ -110,4 +110,4 @@ proxy:
   image:
     repo: vimc
     name: orderly-web-proxy
-    tag: master
+    tag: mrc-4255
```

Comment:
once merged all these tags can be reverted. deployed configs should just work as long as we deploy with `--pull`